include/ultra64/rcp.h

```diff
@@ -454,90 +454,90 @@
 #define VI_X_SCALE_REG      (VI_BASE_REG + 0x30)
 
-// VI y-scale (R/W): [27:16] vertical subpixel offset (2.10 format)
-//                   [11: 0] 1/vertical scale up factor (2.10 format)
+/* VI y-scale (R/W): [27:16] vertical subpixel offset (2.10 format) */
+/*                   [11: 0] 1/vertical scale up factor (2.10 format) */
 #define VI_Y_SCALE_REG      (VI_BASE_REG + 0x34)
 
 /*
  * VI_CONTROL_REG: read bits
  */
-#define VI_CTRL_TYPE_16             0x00002 // [1:0] pixel size: 16 bit
-#define VI_CTRL_TYPE_32             0x00003 // [1:0] pixel size: 32 bit
-#define VI_CTRL_GAMMA_DITHER_ON     0x00004 // 2: default = on
-#define VI_CTRL_GAMMA_ON            0x00008 // 3: default = on
-#define VI_CTRL_DIVOT_ON            0x00010 // 4: default = on
-#define VI_CTRL_SERRATE_ON          0x00040 // 6: on if interlaced
-#define VI_CTRL_ANTIALIAS_MASK      0x00300 // [9:8] anti-alias mode
-#define VI_CTRL_ANTIALIAS_MODE_0    0x00000 // Bit [9:8] anti-alias mode: AA enabled, resampling enabled, always fetch extra lines
-#define VI_CTRL_ANTIALIAS_MODE_1    0x00100 // Bit [9:8] anti-alias mode: AA enabled, resampling enabled, fetch extra lines as-needed
-#define VI_CTRL_ANTIALIAS_MODE_2    0x00200 // Bit [9:8] anti-alias mode: AA disabled, resampling enabled, operate as if everything is covered
-#define VI_CTRL_ANTIALIAS_MODE_3    0x00300 // Bit [9:8] anti-alias mode: AA disabled, resampling disabled, replicate pixels
-#define VI_CTRL_PIXEL_ADV_MASK      0x0F000 // [15:12] pixel advance mode
-#define VI_CTRL_PIXEL_ADV_3         0x03000 // Bit [15:12] pixel advance mode: Always 3 on N64
-#define VI_CTRL_DITHER_FILTER_ON    0x10000 // 16: dither-filter mode
+#define VI_CTRL_TYPE_16             0x00002 /* [1:0] pixel size: 16 bit */
+#define VI_CTRL_TYPE_32             0x00003 /* [1:0] pixel size: 32 bit */
+#define VI_CTRL_GAMMA_DITHER_ON     0x00004 /* 2: default = on */
+#define VI_CTRL_GAMMA_ON            0x00008 /* 3: default = on */
+#define VI_CTRL_DIVOT_ON            0x00010 /* 4: default = on */
+#define VI_CTRL_SERRATE_ON          0x00040 /* 6: on if interlaced */
+#define VI_CTRL_ANTIALIAS_MASK      0x00300 /* [9:8] anti-alias mode */
+#define VI_CTRL_ANTIALIAS_MODE_0    0x00000 /* Bit [9:8] anti-alias mode: AA enabled, resampling enabled, always fetch extra lines */
+#define VI_CTRL_ANTIALIAS_MODE_1    0x00100 /* Bit [9:8] anti-alias mode: AA enabled, resampling enabled, fetch extra lines as-needed */
+#define VI_CTRL_ANTIALIAS_MODE_2    0x00200 /* Bit [9:8] anti-alias mode: AA disabled, resampling enabled, operate as if everything is covered */
+#define VI_CTRL_ANTIALIAS_MODE_3    0x00300 /* Bit [9:8] anti-alias mode: AA disabled, resampling disabled, replicate pixels */
+#define VI_CTRL_PIXEL_ADV_MASK      0x0F000 /* [15:12] pixel advance mode */
+#define VI_CTRL_PIXEL_ADV(n)        (((n) << 12) & VI_CTRL_PIXEL_ADV_MASK) /* Bit [15:12] pixel advance mode: Always 3 on N64 */
```

Comment:
I adjusted this macro while making changes to the comments, I figured it's small enough to get it done now

Reply:
well, it would make importing the rest of the vimodes from ultralib ever-so-slightly harder (it was changed in #2139 if you recall), and it just feels like noise in an already large PR